addOns/network/src/main/java/org/zaproxy/addon/network/ServerCertificatesOptions.java

```diff
@@ -116,7 +120,9 @@ protected void parseImpl() {
             validity = DEFAULT_SERVER_CERT_VALIDITY;
         }
         serverCertValidity = Duration.ofDays(validity);
-        serverCertConfig = new CertConfig(serverCertValidity);
+
+        serverCRLDistributionPoint = getString(SERVER_CERT_CDP, null);
```

Comment:
The acronym in the middle can still follow camelCase. `serverCrlDistributionPoint`